times.go

```diff
@@ -25,28 +25,48 @@ func Time(then time.Time) string {
 	return RelTime(then, time.Now(), "ago", "from now")
 }
 
-var magnitudes = []struct {
+// Magnitude stores one magnitude and the output format for this magnitude of relative time.
+type Magnitude struct {
 	d      int64
 	format string
 	divby  int64
-}{
-	{1, "now", 1},
-	{2, "1 second %s", 1},
-	{Minute, "%d seconds %s", 1},
-	{2 * Minute, "1 minute %s", 1},
-	{Hour, "%d minutes %s", Minute},
-	{2 * Hour, "1 hour %s", 1},
-	{Day, "%d hours %s", Hour},
-	{2 * Day, "1 day %s", 1},
-	{Week, "%d days %s", Day},
-	{2 * Week, "1 week %s", 1},
-	{Month, "%d weeks %s", Week},
-	{2 * Month, "1 month %s", 1},
-	{Year, "%d months %s", Month},
-	{18 * Month, "1 year %s", 1},
-	{2 * Year, "2 years %s", 1},
-	{LongTime, "%d years %s", Year},
-	{math.MaxInt64, "a long while %s", 1},
+}
+
+// NewMagnitude returns a Magnitude object.
+//
+// d is the max number value of relative time for this magnitude.
```

Comment:
The unit of `d` is still not documented. How about:

```Go
// d is the max number value of relative time (in seconds) for this magnitude.
```

Is that right? Including units here is very critical.